<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Show_Summary_Table</class>
 <widget class="QWidget" name="Show_Summary_Table">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{
border-image: url(:/pic/Hotel.png);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QComboBox" name="DayBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 11pt &quot;等线 Light&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>01</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>02</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>03</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>04</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>05</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>06</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>07</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>08</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>09</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>13</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>17</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>18</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>21</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>22</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>23</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>24</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>25</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>26</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>27</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>28</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>29</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>31</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>90</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/pic/Login.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 11pt &quot;等线 Light&quot;;</string>
    </property>
    <property name="text">
     <string>日</string>
    </property>
   </widget>
   <widget class="QComboBox" name="MonthBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 11pt &quot;等线 Light&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>01</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>02</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>03</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>04</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>05</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>06</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>07</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>08</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>09</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 11pt &quot;等线 Light&quot;;</string>
    </property>
    <property name="text">
     <string>月</string>
    </property>
   </widget>
   <widget class="QComboBox" name="YearBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 11pt &quot;等线 Light&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>2016</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="DayBox_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 11pt &quot;等线 Light&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>01</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>02</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>03</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>04</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>05</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>06</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>07</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>08</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>09</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>13</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>17</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>18</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>21</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>22</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>23</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>24</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>25</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>26</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>27</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>28</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>29</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>31</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="MonthBox_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 11pt &quot;等线 Light&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>01</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>02</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>03</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>04</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>05</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>06</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>07</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>08</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>09</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="YearBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 11pt &quot;等线 Light&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>2016</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 11pt &quot;等线 Light&quot;;</string>
    </property>
    <property name="text">
     <string>月</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 11pt &quot;等线 Light&quot;;</string>
    </property>
    <property name="text">
     <string>日</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>431</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="RetBtn">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>340</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/pic/Ret.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
